src/toast/util/toastTypes.ts

```diff
@@ -0,0 +1,25 @@
+export interface ToastData {
+  mode: "danger" | "warning" | "success" | "info";
```

Comment:
Let's not have any `mode` as we don't provide any default UI within Toast component.